sdk/key_vault/src/key.rs

```diff
@@ -0,0 +1,334 @@
+use std::fmt::{Debug, Display};
+
+use azure_core::TokenCredential;
+use chrono::serde::ts_seconds_option;
+use chrono::{DateTime, Utc};
+use getset::Getters;
+use serde::Deserialize;
+use serde_json::{Map, Value};
+use url::Url;
+
+use crate::client::API_VERSION;
+use crate::{KeyVaultClient, KeyVaultError};
+
+/// A KeyBundle consisting of a WebKey plus its attributes.
+#[derive(Debug, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct KeyBundle {
```

Comment:
In other languages, we call this `KeyVaultKey` and has a different shape, probably the closest semantically being Java: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyVaultKey.java

As new models are added, we should strive for consistency (though idiomatic to the language still takes precedent) rather than do it later.

Reply:
Thanks for reviewing! It is good to be consistent, but it's a little strange to me why the actual REST API docs are not the one, final truth. I will make the changes to coincide with the Java SDK, per your request :slightly_smiling_face:

Reply:
We are working to get to a place where the swagger and SDKs are more aligned. This is a much older swagger and names don't conform to normal languages' conventions (in many cases), and structures are sometimes changed based on user experience research. But across languages' SDKs, we try to be consistent when idiomatic differences are insignificant, e.g. Java, JavaScript, and Python might call something `listPropertiesOfSecrets` but .NET doesn't use "list" as a verb, instead using something like `GetPropertiesOfSecrets`.

General guidelines can be found here: https://azure.github.io/azure-sdk/general_introduction.html

Thank you for considering these changes!